python/ray/node.py

```diff
@@ -353,6 +354,16 @@ def merge_resources(env_dict, params_dict):
 
         if not self._resource_spec:
             env_resources = {}
+            cpu_info = get_cpu_info()
+            required_cpu_instruction_str = \
+                os.getenv(
+                    "CPU_INSTRUCTION_SET", ray_constants.CPU_INSTRUCTION_SET)
+            required_cpu_instruction_set = \
+                [x.strip() for x in required_cpu_instruction_str.split(",")]
```

Comment:
Can you handle the exceptions once it fails to parse the sets str?

Reply:
Illegal “sets” does not exist in "cpu_info['flags']"

Reply:
I see. It's hard to handle.